charts/flipt/values.yaml

```
@@ -25,6 +25,7 @@ test:
   # tag: '1.36.1'
 
 podAnnotations: {}
+podLabels: {"blah": "blah"}
```

I believe these are the defaults, so best to keep this as the empty set.
```suggestion
podLabels: {}
```

---

Oops, I did not mean to commit that!